src/utils/transaction.ts

```diff
@@ -717,49 +628,30 @@ export const buildCrossChainAction = async (
             amount: amountBN.toString(),
             iconUrl: fromAssetIconUrl,
           },
-          amount: BigNumber.from(routeData?.bestRoute?.toAmount),
+          amount: 1,
           toAsset: {
             address: plrDaoAsset.address,
-            decimals: 9,
-            symbol: 'PillarDAO',
-            amount: result.result?.returnAmount.toString() ?? '0',
+            decimals: toAssetDecimals,
+            symbol: 'Pillar DAO NFT',
+            amount: '1',
             iconUrl: 'https://public.pillar.fi/files/pillar-dao-member-badge.png',
           },
           receiverAddress: transactionBlock?.values?.receiverAddress,
-          providerName: result.result?.provider
-            ? swapServiceIdToDetails[result.result.provider].title
-            : 'Unknown provider',
-          providerIconUrl: result.result?.provider ? swapServiceIdToDetails[result.result.provider].iconUrl : '',
+          providerName: '',
+          providerIconUrl: '',
         };
-
         const crossChainAction: ICrossChainAction = {
           id: crossChainActionId,
           relatedTransactionBlockId: transactionBlock.id,
           chainId: fromChainId,
           type: TRANSACTION_BLOCK_TYPE.PLR_DAO_STAKE,
           preview,
-          transactions: routeData.transactions || [],
+          transactions,
           isEstimating: false,
           estimated: null,
-          containsSwitchChain: routeData?.bestRoute?.containsSwitchChain,
-          bridgeUsed: routeData?.bestRoute?.steps[0].tool,
-          receiveAmount: routeData?.bestRoute?.toAmount,
+          receiveAmount: '1',
```

Comment:
always 1?

Reply:
Yes it will be always 1 PLR DAO NFT.

Reply:
not sure if it's needed then at all here, but if you think it should be kept let's leave it